flexget/plugins/input/next_series_seasons.py

```diff
@@ -115,6 +115,19 @@ def on_task_input(self, task, config):
                 else:
                     latest_season = low_season + 1
 
+                if (latest_season - low_season > 15 and not series.begin) or (series.begin and
+                    latest_season - series.begin.season > 30):
+                    if series.begin:
+                        log.error('Series `%s` has a begin episode set (`%s`), but the season currently being processed '
+                                  '(%s) is %s seasons later than it. To prevent emitting incorrect seasons, this ' 
+                                  'series will not emit unless the begin episode is adjusted to a season that is less '
+                                  'than 30 seasons from season %s.', series.name, series.begin.identifier, latest_season,
+                                  (latest_season - series.begin.season), latest_season)
+                    else:
+                        log.error('Series `%s` does not have a begin episode set and continuing this task would result '                                   'in more than 15 seasons being emitted. To prevent emitting incorrect seasons, this '
+                                  'series will not emit unless the begin episode is set in your series config or by '
+                                  'using the CLI subcommand `series begin "%s" <SxxExx>`.', series.name, series.name)
+                    continue
```

Comment:
No, this is outside the loop that counts through the seasons. This should only happen once per series.